src/main/java/buildcraftAdditions/blocks/multiBlocks/PaternTestingBlock.java

```diff
@@ -25,4 +27,8 @@ public PaternTestingBlock() {
 	public void onBlockPlacedBy(World world, int x, int y, int z, EntityLivingBase entity, ItemStack stack) {
 		TestingPatern.build(new Location(world, x, y, z));
 	}
+
+	@Override
+	public void registerBlockIcons(IIconRegister register) {
```

Comment:
To prevent the exception shown in the console that the item icon couldnt be found.